CalibPPS/TimingCalibration/plugins/PPSTimingCalibrationPCLHarvester.cc

```diff
@@ -0,0 +1,160 @@
+/****************************************************************************
+ *
+ * This is a part of PPS offline software.
+ * Authors:
+ *   Edoardo Bossini
+ *   Piotr Maciej Cwiklicki
+ *   Laurent Forthomme
+ *
+ ****************************************************************************/
+
+#include "DQMServices/Core/interface/DQMEDHarvester.h"
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+
+#include "Geometry/VeryForwardGeometryBuilder/interface/CTPPSGeometry.h"
+#include "Geometry/Records/interface/VeryForwardRealGeometryRecord.h"
+
+#include "CalibPPS/TimingCalibration/interface/TimingCalibrationStruct.h"
+#include "CondCore/DBOutputService/interface/PoolDBOutputService.h"
+
+#include "DataFormats/CTPPSDetId/interface/CTPPSDiamondDetId.h"
+#include "CondFormats/PPSObjects/interface/PPSTimingCalibration.h"
+
+#include "TProfile.h"
+#include "TF1.h"
```

Comment:
```suggestion
#include <TProfile.h>
#include <TF1.h>
```

however, are they really needed ? from a quick look the DQM should be enough.
So
```suggestion
```

Reply:
Many thanks for your detailed review, @fwyzard ! Beside the possible improvement of this HLT filter fillDescriptions member, this should all be fixed in the latest push.